src/main/java/com/zendesk/maxwell/schema/columndef/DecimalColumnDef.java

```diff
@@ -21,4 +21,9 @@ public String toSQL(Object value) {
 		return d.toEngineeringString();
 	}
 
+	@Override
+	public ColumnDef copy() {
+		return new DecimalColumnDef(name, type, pos);
+	}
```

Comment:
you mean repeating this endlessly?  Yeah, I mean you can play horrible tricks with reflection, but as with most things in javaland it's better to be verbose and clear than terse and horrible